web-client/terraform/main/main.tf

```diff
@@ -122,3 +122,14 @@ resource "aws_acm_certificate_validation" "dns_validation_west" {
   validation_record_fqdns = [aws_route53_record.record_certs.fqdn]
   provider                = aws.us-west-1
 }
+
+resource "aws_route53_record" "statuspage" {
+  count   = var.statuspage_dns_record != "" ? 1 : 0
```

Comment:
neat use of count!

Reply:
yea i was trying to figure out how to make this optional. looks like this worked as expected. pretty cool. thanks for the nudge in the right direction @adunkman